mod-users/src/main/resources/domain/mod-users/features/util/initData.feature

```diff
@@ -0,0 +1,30 @@
+Feature: init data for mod-users
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+    * def patronId = call uuid1
+    * def patronName = call random_string
+    * def barCode = call random_numbers
+
+  @PostPatronGroupAndUser
+  Scenario: create PatronGroup & User
+    * def createPatronGroupRequest = read('samples/PatronGroup/create-patronGroup-request.json')
```

Comment:
I thought @steveellis suggested in a previous pull request that these kinds of uses are better inlined in the code.

@steveellis Am I mis-recollecting feedback you gave previously?

Reply:
we need an **USER** for almost every test case, so writing multiple lines of same code for each test case is i don't think good way .
we have annotation facility also in karate to reduce the lines of code.
 Let me know if any suggestion on this.

Reply:
> we need an **USER** for almost every test case

I haven't questioned the need at all.

> so writing multiple lines of same code for each test case is i don't think good way 

Isn't this file already a definition of shared functions that are invoked from other places or have I misinterpreted what this is?

> we have annotation facility also in karate to reduce the lines of code. Let me know if any suggestion on this.

What annotation facility are you referring to?

Reply:
I'm closing this following a conversation we had privately. 

I don't think we are going to reach an agreement on how to define the JSON in these tests.